<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loading app</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="more_info_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 220);
border-radius: 20px;
border: 5px solid rgb(0, 92, 157)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="loading_progress_status">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Please wait....</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="my_progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	background-color:rgb(43, 31, 91);
	border-style: none;
	border-radius: 10px;
	text-align: center;
	color: rgb(255, 0, 0)
}
QProgressBar::chunk{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 136, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius: 10px;
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
